charts/kubearchive/values.yaml

```diff
@@ -93,3 +88,13 @@ database:
     dbUser: "ps_user"
     # database password
     dbPassword: "P0stgr3sdbP@ssword"  # notsecret
+
+observability:
+  # If "true" a Jaeger all-in-one Deployment is created in the '{{ .Release.Namespace }}' namespace
+  enabled: false
```

Comment:
If it's enabled here, then shouldn't the `enabled` keys on the individual components be removed as well?  I don't see the point in having them in both places, and I think it should be on or off for all of k9e, not on a component-by-component basis.

Reply:
If I understood you correctly, they are already removed. See https://github.com/kubearchive/kubearchive/pull/214/files#diff-860af8aee46eeef818486b58498628115cc3a9427545ea9e0f32b90d4b7e5089L50 and https://github.com/kubearchive/kubearchive/pull/214/files#diff-860af8aee46eeef818486b58498628115cc3a9427545ea9e0f32b90d4b7e5089L28